packages/vaadin-material-styles/user-colors.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './version.js';
+import { css } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+const userColors = css`
+  :host {
+    --vaadin-user-color-0: #ab47bc;
```

Comment:
See the original Material styles [here](https://github.com/vaadin/collaboration-engine-internal/blob/2ed178db85f4a911b440ca66c17e2772fcd5206f/collaboration-engine-frontend/field-highlighter/theme/material/vaadin-user-colors.js#L8).